routes/web/project/location/postgres.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class CloverWeb
+  hash_branch(:project_location_prefix, "postgres") do |r|
+    Authorization.authorize(@current_user.id, "Project:view", @project.id)
```

Comment:
Do you believe the user requires 'Project:view' permission to access project resources? If so, it would be good to check it at https://github.com/ubicloud/ubicloud/blob/a5558811d2e91c1fa05c6ed2ee9a71bafd902ae4/routes/web/project.rb#L27

Reply:
I think yes. Also we access to the project in next line when we check if enable_postgres is set.

Without Project:view check, for a project which user doesn't have access if enable_postgres is not set, we return Not Found but with this check we return Forbidden. So without the check we leak information.